src/main/java/com/demo/MybatisApplication/dto/StudentDisplayByIdDto.java

```diff
@@ -0,0 +1,11 @@
+package com.demo.MybatisApplication.dto;
+
+import lombok.Data;
+
+@Data
+public class StudentDisplayByIdDto {
```

Comment:
```suggestion
public class StudentDisplayDto {
```
The name should be generic. This DTO can be used in other places too and not just for the byId method.

Reply:
Done